src/cogent3/core/alignment.py

```diff
@@ -2013,6 +2013,22 @@ def __init__(self, map, data, length=None):
         if hasattr(data, "name"):
             self.name = data.name
 
+    def annotate_matches_to(
+        self, pattern=None, annot_type=None, name=None, allow_multiple=False
+    ):
+        """Creates an annotation at the specified pattern in a sequence."""
```

Comment:
change "create" to "add". Plus, add a "Returns" section, "returns a list of Annotation instances"